ui/app/components/empty-state.js

```diff
@@ -0,0 +1,6 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  title: null,
+  classNames: ['empty-state box is-sideless is-marginless is-fullwidth is-bottomless'],
```

Comment:
Since we're styling up this component, do we want to rely on the Bulma classes? Or should we just drop those for new components and start building out our own primitives? (I'm not sure I have an answer now, but as we build out new things it seems like a good time to split from Bulma if it's not too painful).